src/utils.rs

```diff
@@ -66,14 +69,24 @@ pub fn extra_offset(text: &str, offset: Indent) -> usize {
     }
 }
 
+// Uses Cow to avoid allocating in the common cases.
 #[inline]
```

Comment:
This function isn't that trivial any more. Probably should leave it to the compiler to decide on inlining.